generators/PleadingKitten/generate.ts

```diff
@@ -0,0 +1,52 @@
+import { GeneratorFunction } from '../../types/GeneratorTypes';
+import { MultilineText } from '../../utils/MultilineText';
+import { loadImage } from '../../utils/loadImage';
+
+import { buildAltText } from './buildAltText';
+import { CANVAS_HEIGHT, CANVAS_WIDTH, FONT_SIZE } from './constants';
+import { PleadingKittenSettings } from './types';
+
+export const generate: GeneratorFunction<PleadingKittenSettings> = async (
+  canvas,
+  settings
+) => {
+  const { topCatLabel, bottomCatLabel } = settings;
+
+  canvas.width = CANVAS_WIDTH;
+  canvas.height = CANVAS_HEIGHT;
+
+  const ctx = canvas.getContext('2d');
+  if (!ctx) {
+    return {
+      success: false,
+    };
+  }
+
+  const image = await loadImage('/assets/pleading-kitten.jpg');
+  ctx.drawImage(image, 0, 0);
```

Comment:
If you use the `setupCanvas` function you can apply the canvas size and draw the background image in one go :)